<launch>

  <arg name="model" default="burger"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find multi_fht_map)/rviz/fhtmap1.rviz" />


  <node name="map_building" pkg="multi_fht_map" type="robot_multi.py" output="screen">
    <param name="network" value="retrievalSfM120k-vgg16-gem" />
    <param name="platform" value="cuda" />
    <param name="robot_name" value="robot1"/>
    <param name="robot_num" value="1"/>
  </node>

  <node name="experiment" pkg="multi_fht_map" type="experiment.py" output="screen">
    <param name="robot_num" value="1"/>
  </node>

</launch>